src/modules/blank-target-link.js

```diff
@@ -27,6 +27,7 @@
 				if (!!href && (/^https?:\/\//.test(href) || /^\/workspace/.test(href))) {
 					if (!t.attr('target')) {
 						t.attr('target', '_blank');
+						t.attr('rel', 'noopener');
 					}
```

Comment:
`if (!!href && (/^https?:\/\//.test(href) || /^\/workspace/.test(href))) {
					if (!t.attr('target')) {
						t.attr('target', '_blank');
						t.attr('rel', 'noopener');
					}
				}`

On the link match or the other condition if there's no target attr set? @nitriques